### DISCUSSION TITLE
How do i communicate with the server in local?

### DISCUSSION THREAD

**Discussion by koinetop** [oobabooga/text-generation-webui#1099] [General]
Hello
Thank you to the developers for the product.
Can you please tell me how to contact the server from a home network, from another computer?
127/0/0/1:7860 does not work
I added a line to the webui bat:
set COMMANDLINE_ARGS=--listen
but  it doesn't work at 192.168.xxx.xxx:7860 ((

tnx

**Comment by mcmonkey4eva** [comment]
Check if the program's been blocked by a firewall
